specs/flyout/flyout.md

```diff
@@ -0,0 +1,179 @@
+# Popover (aka Flyout)
+## Overview
+Popover is a focusable floating container that shows over a pages content and displays UI related to what the user is doing. It is typically triggered by a clickable element. Similar to a Dialog, but with positioning controls, no overlay option, and no specific action being required, a Popover can be used to reveal a secondary control or show more detail about an item.
```

Comment:
> with positioning controls

Are the positioning controls part of the UI or are they part of the API?

Reply:
They are part of the API